src/features/ci-cd/hooks/track-latest-release.ts

```diff
@@ -2,19 +2,18 @@ import * as _ from 'lodash';
 
 import { sbvrUtils, hooks } from '@balena/pinejs';
 
-const updateLatestRelease = async (
-	id: number,
-	{ request, api }: hooks.HookArgs,
-) => {
-	// We only track builds that are successful
-	if (request.values.status !== 'success') {
```

Comment:
@Page- this is the only part of this PR that I'm troubled by -- this approach using params from the `request` date back to the split of open and closed APIs. I don't see any reason this approach should be different (other than not short circuiting as frequently), but i wanted to make sure I understood the approach before flipping this from draft.

Reply:
Yeah the new approach just seems like it has less short-circuiting but that makes sense since it's now possible for "track latest" to trigger a downgrade if the previously newest version becomes invalid